torch/csrc/jit/operator.cpp

```diff
@@ -160,6 +160,15 @@ struct SchemaParser {
       L.next();
       value = DynamicType::get();
       alias_info = parseAliasAnnotation();
+    } else if (L.cur().kind == TK_IDENT && L.cur().text() == "Dict") {
+      L.next();
+      L.expect('[');
```

Comment:
discussion: change to '(' to match future. Try to see if we can normalize syntax.